src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -117,7 +120,7 @@ private void EmbedIfNeedTo(BoundExpression receiver, ImmutableArray<PropertySymb
             // If we are calling a method on a NoPIA type, we need to embed all methods/properties
             // with the matching name of this dynamic invocation.
             var module = this.EmitModule;
-            if (module != null && receiver != null && (object)receiver.Type != null)
+            if (module != null && receiver != null && receiver.Type is { })
```

Comment:
Should `receiver` be annotated?

Reply:
As far as I can tell no caller can pass a null value.

---
In reply to: [389200135](https://github.com/dotnet/roslyn/pull/42068#discussion_r389200135) [](ancestors = 389200135)